com.sweetscape.ZeroOneZeroEditor.yml

```
@@ -0,0 +1,57 @@
+app-id: com.sweetscape.ZeroOneZeroEditor
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: 010editor
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=home
+
+modules:
+  - name: perl-libs
+    buildsystem: simple
+    build-commands:
+      - 'perl-libs/install.sh'
+    post-install:
+      - 'chmod -Rf u+w /app/lib/perl5/site_perl'
+    sources:
+      - generated-sources.json
+
+  - name: unpack-install-jammer
+    buildsystem: simple
+    build-commands:
+      - chmod +x extract.pl
+      - mkdir -p /app/bin
+      - mv extract.pl /app/bin/extract.pl
+    sources:
+      - type: git
+        url: https://github.com/lod/unpack-install-jammer.git
+        tag: 0.2.0
+        commit: 41faa5a8c7925db4ff7ca665cca478979d177de4
+
+  - name: 010editor
+    only-arches: x86_64
```

```suggestion
    only-arches: [x86_64]
```

also you need to add a flathub.json

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64